guides/source/security.md

```diff
@@ -1212,6 +1212,14 @@ Rails.application.config.content_security_policy do |policy|
 end
 ```
 
+Enable the [`report-to`][] directive to report violations to the specified Reporting API group:
```

Comment:
This is missing the link - if you look at line 1239 below you can see how the other links in these docs are handled